package.json

```diff
@@ -105,9 +105,10 @@
     "loud-rejection": "^1.2.0",
     "max-timeout": "^1.0.0",
     "md5-hex": "^1.2.0",
-    "meow": "^3.6.0",
+    "meow": "^3.7.0",
```

Comment:
Actually, it's require for this PR to work. https://github.com/sindresorhus/meow/pull/25